packages/cli/src/config.ts

```diff
@@ -46,6 +46,8 @@ export function getTypeScriptConfig(path: string, system = sys) {
     config,
     system,
     dirname(path),
+    undefined,
+    path,
```

Comment:
?

Reply:
`Program.getResolvedProjectReferences` requires the path to be set for the config file.